webpack.config.js

```diff
@@ -42,7 +42,10 @@ const config = {
         rules: [
             {
                 test: /\.jsx?$/,
-                include: path.resolve(__dirname, 'assets'),
+                include: [
+                    path.resolve(__dirname, 'assets'),
+                    path.resolve(__dirname, 'node_modules/bootstrap')
```

Comment:
why we need this?

Reply:
The module files for Bootstrap include a spread operator. Without compilation, these fail

Reply:
I see

Reply:
Hopefully with removing jQuery, and importing classes like Tooltip directly may fix this. Will look into it tomorrow

Reply:
@petrjasek Specifically importing the Bootstrap modules directly did not work, it is still importing the dist files that include the spread operator.

This project to upgrade to Bootstrap v5 also gives us the opportunity to upgrade other packages.
* Babel v7 / preset-env (and target the last x browser versions)
* Webpack v5

This might fix the issue with Bootstrap v5, as newer Browsers support the spread operator (and Bootstrap v5 no longer supports older Browsers).
What are your thoughts?